force-app/main/default/classes/TB_GenericHttpRelayClient.cls

```diff
@@ -29,6 +29,7 @@ public class TB_GenericHttpRelayClient implements TB_IOutboxRelayClient {
         req.setMethod(method);
         Http http = new Http();
         HTTPResponse res = http.send(req);
+        ctx.setResult(new GenericRelayResult(res));
```

Comment:
Added a `result` to the RelayContext which can be used to store information about the relay response